less/index.less

```
@@ -57,3 +57,152 @@ html, body {
     height: 100%;
 }
 
+// Resume code starts here
+
+// Variables
+@main-color: black;
+@content-font-size: 1rem;
+@sub-header-size: 1.5rem;
+@title-font-size: 1.8rem;
+@h1-font: 2.25rem;
+
+// Mixins
+.flex-even {
+	display: flex;
+	justify-content: space-evenly;
+}
+.media-query-size {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+
+.container {
+	width: 100%;
+	margin: 0 auto;
+}
+// Header
+
+header nav {
+	width: 100%;
+	.flex-even;
+	flex-direction: row;
+	margin-bottom: 4rem;
+	background: @main-color;
+
+	@media(max-width: 500px) {
+		width: 100%;
+		display: flex;
+		flex-direction: column;
+	  }
+
+	a {
+		font-size: @title-font-size;
+		color: @main-color;
+		text-align: center;
+		justify-content: center;
+		text-decoration: none;
+		margin: 30px 30px;
+		color: white;
+	}
+}
+// intro content
+
+.container .title-img-card {
+	.flex-even;
+	border: 2px solid black;
+	width: 90%;
+	margin: 30px 0px;
+	margin-bottom: 50px;
+	margin: auto;
```

```suggestion
	margin: auto;
	flex-wrap: wrap;
```
Maybe add a wrap here? The text overlays the image as the screen shrinks